scripts/install_third_party.sh

```diff
@@ -66,73 +66,57 @@ fi
 #
 $NPM_CMD config set ca ""
 
-echo Checking whether jsrepl is installed in third_party
-if [ ! "$NO_JSREPL" -a ! -d "$THIRD_PARTY_DIR/static/jsrepl" ]; then
-  echo Installing CoffeeScript
-  $NPM_INSTALL coffee-script@1.2.0 ||
-  {
-    echo ""
-    echo "  [oppia-message]"
-    echo ""
-    echo "  Instructions:"
-    echo "    If the script fails here, please try running these commands (you"
-    echo "    may need to use sudo):"
-    echo ""
-    echo "      chown -R $ME ~/.npm"
-    echo "      rm -rf ~/tmp"
-    echo ""
-    echo "    Then run the script again."
-    echo ""
-    echo "  What is happening:"
-    echo "    npm, a package manager that Oppia uses to install some of its"
-    echo "    dependencies, is putting things into the ~/tmp and ~/.npm"
-    echo "    folders, and then encounters issues with permissions."
-    echo ""
-    echo "  More information:"
-    echo "    http://stackoverflow.com/questions/16151018/npm-throws-error-without-sudo"
-    echo "    https://github.com/isaacs/npm/issues/3664"
-    echo "    https://github.com/isaacs/npm/issues/2952"
-    echo ""
-
-    exit 1
-  }
-
-  echo Installing uglify
-  $NPM_INSTALL uglify-js
-
-  if [ ! -d "$TOOLS_DIR/jsrepl/build" ]; then
-    echo Downloading jsrepl
+# Download and install Skulpt.
+
+# GitPython is needed by Skulpt's dist build function. Download a local copy to
+# avoid the user needing to install it.
+echo Checking whether GitPython is installed in third_party
```

Comment:
This was the approach I was trying at first (ie, including Skulpt's skulpt.py script into install_third_party.py's syspath, then manually calling its dist() function. Overall, it seemed convoluted and more complicated than what was done here. I really wanted to avoid using GitPython, too, but that would have involved rebuilding most of the dist-build process of Skulpt, which seemed like a non-portable solution to me.